src/DocsGenerator/Program.cs

```diff
@@ -29,12 +29,9 @@ public static int Main(string[] args)
                 File.Copy(file, destFile, true);
             }
 
-
-            var tsqllintVersion = XDocument.Load(@"tsqllint/source/TSQLLint.Console/TSQLLint.Console.csproj").Root

```

Comment:
We should be reading the version from the file and not hardcoding it, it will go out of sync quickly.

On the other project we have something like:
```
var sonarVersion = File.ReadAllText(".SONAR_VERSION")
                .TrimEnd(Environment.NewLine.ToCharArray())
                .Split('.')
```

we can try doing the same.